### ISSUE TITLE
maxfile does not upload .pdf for sharing

### ISSUE THREAD

**Issue by gotquestions** [nokonoko/Pomf#39]
maxfile.ro does not upload .pdf
It just displays it but does not upload for sharing.
How can I share a .pdf please?
Thankyou

**Comment by SEAPUNK** [comment]
This is something you'd probably have to ask the owners of Maxfile. Even though it's running on this software, it's still using their setup.

**Comment by gotquestions** [comment]
Okay thankyou for response. 
There is no support forum help from Maxfile, there is only a link to this github site.
But Maxfile responds here sometimes so I will check back on this thread to see if there is any reply from Maxfile here hopefully.
Thankyou SEAPUNK.

**Comment by avail** [comment]
You should create an issue on Maxfile's fork of Pomf.
https://github.com/alin1337/Pomf

**Comment by gotquestions** [comment]
Okay I will do that thankyou very much **avail**

**Comment by gotquestions** [comment]
thankyou but there is no issues permitted there. I see no issues button link there at
https://github.com/alin1337/Pomf

**Comment by alucard0134** [comment]
This impossible to change on a github repo. Your going to have to contact the maintainer of that site to change it.

tl;dr we can fix this because it ain't our fault
